spec/student_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './student'
+
+describe Student do
+  before :each do
+    @student = Student.new 17, 'Francisco'
+  end
+  describe '#new' do
+    it 'Return a new Student object' do
+      expect(@student).to be_an_instance_of Student
+    end
```

Comment:
- [ ] Let ensure that there are unit tests for all the methods in all the classes, please kindly write unit test for the `add_classroom` method in the Student class.